app/src/utils/restRequest/go.ts

```diff
@@ -44,6 +45,7 @@ export interface AdditionalOptions {
     isCsvRequest?: boolean;
     enforceEnglishForQuery?: boolean;
     useCurrentLanguageForMutation?: boolean;
+    isExcelRequest?: boolean;
```

Comment:
Let's create a general `contentType` prop and replace `isCsvRequest` and `isExcelRequest`